### ISSUE TITLE
number of notifications in the tabs

### ISSUE THREAD

**Issue by demarco5** [Bl4Cc4t/GoodTwitter2#281]
The old twitter if you made a search and left that tab the tab would still auto pull the latest results and put the number (1) or more in the tab so we knew to click back on that tab to fetch those latest tweets

is this possible still?

**Comment by Bl4Cc4t** [comment]
I don't think it is. The site still refreshes if you scroll all the way up again, but there is no way to determine the number of new tweets that are to be shown. This probably cannot be done.

**Comment by demarco5** [comment]
yeah. even if it showed any notification number to know theres new tweets.  Ah, what a shame they got rid of this, thanks for the reply

**Comment by Bl4Cc4t** [comment]
Yeah, Twitter's decisions nowadays are rather strange to be honest. The whole concept of having a mobile layout on desktop is the very reason this project exists... but most people do not seem to be bothered by this for some reason.
I'll keep this open in case someone discovers a way to do this.

**Comment by demarco5** [comment]
yeah this was my main issue last summer, good twitter obviously fixed it but then they depricated that api and now i guess theres no options for this, it was their best feature in my opinion, so annoying. appreciate  the quick replies, thanks
